### ISSUE TITLE
Swiping sidebar from right on frontpage

### ISSUE THREAD

**Issue by l3d00m** [ccrama/Slide#96]
## Info
Opens the sidebar for /r/android for me and then crashes. Using single mode and newest build. 
## Stacktrace
```java.lang.RuntimeException: Unable to destroy activity {me.ccrama.redditslide/me.ccrama.redditslide.Activities.CommentsScreen}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object java.util.ArrayList.get(int)' on a null object reference
at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:3763)
at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:3781)
at android.app.ActivityThread.access$1400(ActivityThread.java:151)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1363)
at android.os.Handler.disp
atchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:135)
at android.app.ActivityThread.main(ActivityThread.java:5349)
at java.lang.reflect.Method.invoke(N
ative Method)
at java.lang.reflect.Method.invoke(Method.java:372)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:908)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:703)Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object java.util.ArrayList.get(int)' on a null object reference
at me.ccrama.redditslide.Fragments.CommentPage.onCre
ateView(CommentPage.java:182)
at android.support.v4.app.Fragment.performCre
ateView(Fragment.java:1965)
at android.support.v4.app.FragmentManagerImpl.moveToSt
ate(FragmentManager.java:1078)
at android.support.v4.app.FragmentManagerImpl.moveToSt
ate(FragmentManager.java:1259)
at android.support.v4.app.FragmentManagerImpl.moveToSt
ate(FragmentManager.java:1241)
at android.support.v4.app.FragmentManagerImpl.disp
atchReallyStop(FragmentManager.java:2080)
at android.support.v4.app.FragmentController.disp
atchReallyStop(FragmentController.java:213)
at android.support.v4.app.FragmentActivity.onReallyStop(FragmentActivity.java:749)
at android.support.v4.app.FragmentActivity.doReallyStop(FragmentActivity.java:735)
at android.support.v4.app.FragmentActivity.onDestroy(FragmentActivity.java:324)
at android.support.v7.app.AppComp
atActivity.onDestroy(AppComp
atActivity.java:161)
at android.app.Activity.performDestroy(Activity.java:6199)
at android.app.Instrument
ation.callActivityOnDestroy(Instrument
ation.java:1140)
at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:3750)... 10 more```
Reported via Slide Crash Capture

**Comment by l3d00m** [comment]
I cannot reproduce the crash a second time, but sidebar for frontpage is still /r/androids frontpage.

**Comment by Alexendoo** [comment]
Yeah that's due to the placeholder text being set as /r/android in [subinfo.xml](https://github.com/ccrama/Slide/blob/a70e3764093a1fc37e257ab81fe921e3bfe2ebcd/app/src/main/res/layout/subinfo.xml)

**Comment by ccrama** [comment]
Has been fixed with 824b092117ae42d3759f65f86ab984530f3d0965

**Comment by l3d00m** [comment]
Thanks for the awesome and really fast support
